server/lib/config/pbench-server.cfg.example

```diff
@@ -38,6 +38,12 @@ port = 0000
 host = graphql.example.com
 port = 0000
 
+[authentication]
```

Comment:
Can you add a comment block documenting this section, and comments for each of the settings to describe what they mean and why you'd might want to specify values?

Reply:
Addressed.

Reply:
Shouldn't line 41 have only one `#` at the beginning?

Reply:
I looked at the line 9 and followed it :smile:

Reply:
Ah, yes, well...where are your lines of `#`'s before an after it, then??  😉